test/plugins/validation/2and3/schema-ibm.js

```diff
@@ -984,12 +867,6 @@ describe('validation plugin - semantic - schema-ibm - OpenAPI 3', () => {
   });
 
   it('should process allOf, oneOf, anyOf schemas correctly', () => {
-    const config = {
-      schemas: {
-        invalid_type_format_pair: 'error'
-      }
-    };
-
```

Comment:
All the changes above here appear to be unrelated to this feature.

Reply:
Yes, why I was introducing tests to this module, I thought it would be a good idea to clean up the individual declarations of config and replace them with one import. I changed the config name to customConfig for the cases where the test uses a custom value.

Reply:
^ I support these changes